charts/rancher-logging/0.0.1/charts/fluentd-tester/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+name: fluentd-tester
```

Comment:
It seems `fluentd-tester` chart is a child collection of `fluentd`, I think you can merge them together, then use a switch or some commands to control.

Reply:
The fluentd is ref from helm/charts, fluentd-elasticsearch, I prefer separate, then it could be upgraded separately.